packages/openactive-integration-tests/test/features/core/common-error-conditions/implemented/unknown-endpoint-test.js

```diff
@@ -0,0 +1,35 @@
+const { expect } = require('chai');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const RequestHelper = require('../../../../helpers/request-helper');
+
+const { BOOKING_API_BASE } = global;
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'core',
+  testFeature: 'common-error-conditions',
+  testFeatureImplemented: true,
+  testIdentifier: 'unknown-endpoint',
+  testName: 'Expect an UnknownOrIncorrectEndpointError for requests to unknown endpoints',
+  testDescription: 'Send a request to an endpoint that does not exist, and expect an UnknownOrIncorrectEndpointError to be returned',
+  numOpportunitiesUsedPerCriteria: 0,
+},
+(configuration, orderItemCriteria, featureIsImplemented, logger) => {
+  describe('Unknown Endpoint - JSON PUT', () => {
+    it('should return an UnknownOrIncorrectEndpointError', async () => {
```

Comment:
Same comment as https://github.com/openactive/openactive-test-suite/pull/172#discussion_r484826391, is this reusable?